base/src/main/java/io/fabric8/launcher/base/identity/RSAPublicKeyConverter.java

```diff
@@ -0,0 +1,56 @@
+package io.fabric8.launcher.base.identity;
+
+import java.math.BigInteger;
+import java.security.KeyFactory;
+import java.security.NoSuchAlgorithmException;
+import java.security.interfaces.RSAPublicKey;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.RSAPublicKeySpec;
+import java.security.spec.X509EncodedKeySpec;
+
+import org.apache.commons.codec.binary.Base64;
+
+public class RSAPublicKeyConverter {
+
+    public static final String PUBLIC_KEY_ALGORITHM = "RSA";
+
+    private static final KeyFactory kf;
+
+    static {
+        try {
+            kf = KeyFactory.getInstance(PUBLIC_KEY_ALGORITHM);
+        } catch (NoSuchAlgorithmException e) {
+            throw new RuntimeException(e);
```

Comment:
may be good to define and throw a dedicated exception instead of using runtimeexception, exception, error, etc..

Reply:
Not sure really, this should create key factory, as key alg is built-in RSA. Maybe

```suggestion
throw new IllegalStateException("Something went terribly wrong here, cannot create key factory for " + PUBLIC_KEY_ALGORITHM, e);
```

WDYT?

Reply:
looks good!